recipes/simpeg-drivers/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "simpeg-drivers" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MiraGeoscience/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 4a23457f414da43b67324a9d11dfcff1d18ae33fc2f7bf2e22af427d0ed4750b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10, <3.11
+    - poetry-core >=1.0.0
+    - pip
+  run:
+    - python >=3.10,<3.11
+    - dask-core 2022.10.*
+    - discretize >=0.10.0,<0.11.0
+    - distributed 2022.10.*
+    - numpy >=1.23.5,<1.24.0
+    - scipy >=1.10.0,<1.11.0
+    - zarr >=2.14.2,<2.15.0
+
+    - geoh5py >=0.9.0,<0.10.0
+    - octree-creation-app >=0.1.1,<0.2.0
+    - geoapps-utils >=0.3.0,<0.4.0
+    - mira-simpeg >=0.19.0.8,<0.19.0.9
+    - param-sweeps >=0.1.8,<0.1.9
+
+    - pillow >=10.3.0,<10.4.0
+    - fsspec 2022.*
+    - geoana >=0.4.0,<0.5.0
+    - h5py >=3.2.1,<4.0.0
+    - matplotlib-base >=3.7.1,<3.8.0
+    - mkl 2022.1.*
+    - pandas >=2.2.1,<2.3.0
+    - pydantic >=2.5.2,<2.6.0
+    - pymatsolver >=0.2.0,<0.3.0
+    - scikit-learn >=1.4.0,<1.5.0
+    - tqdm >=4.66.1,<5.0.0
+    - tbb 2021.12.*
+    - python-tzdata 2023.4.*
```

Comment:
**question:** does it need to repeat indirect dependencies, such as h5py which its already required by geoh5py?